apps/challenges/views.py

```diff
@@ -423,6 +424,10 @@ def get_all_challenges(request, challenge_time):
     # make sure that a valid url is requested.
     if challenge_time.lower() not in ("all", "future", "past", "present"):
         response_data = {"error": "Wrong url pattern!"}
+        Metrics.total_http_requests_sample.labels(
+            "challenge", "get_all_challenges", request.method, "406"
```

Comment:
@burnerlee we need to move this to a utils method which will accept `app_label` `api_label` `request_type` and `status`